src/passes/builtinHandler/utils/unckeckedMathUtils.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  BinaryOperation,
+  FixedBytesType,
+  FunctionCall,
+  FunctionCallKind,
+  Identifier,
+  IntType,
+} from 'solc-typed-ast';
+import { AST } from '../../../ast/ast';
+import { typeNameFromTypeNode } from '../../../utils/utils';
+import { safeGetNodeType } from '../../../utils/nodeTypeProcessing';
+import assert from 'assert';
+import { printNode, printTypeNode } from '../../../utils/astPrinter';
+import { getIntOrFixedByteBitWidth } from '../../../warplib/utils';
+
+export function functionaliseUncheckedAdd(node: BinaryOperation, ast: AST) {
+  IntxIntUncheckedFunction(node, 'add', ast);
+}
+
+export function functionaliseUncheckedSub(node: BinaryOperation, ast: AST) {
+  IntxIntUncheckedFunction(node, 'sub', ast);
+}
+
+function IntxIntUncheckedFunction(node: BinaryOperation, name: string, ast: AST) {
+  const lhsType = typeNameFromTypeNode(safeGetNodeType(node.vLeftExpression, ast.inference), ast);
+  const rhsType = typeNameFromTypeNode(safeGetNodeType(node.vRightExpression, ast.inference), ast);
+  const retType = safeGetNodeType(node, ast.inference);
+  assert(
+    retType instanceof IntType || retType instanceof FixedBytesType,
+    `${printNode(node)} has type ${printTypeNode(retType)}, which is not compatible with ${name}`,
+  );
+  const width = getIntOrFixedByteBitWidth(retType);
+
+  const fullName = `u${width}_overflow${width === 256 && name === 'sub' ? '' : `ing`}_${name}`;
+
+  const importName = ['integer'];
```

Comment:
Let's make a function in `importPaths.ts` which returns the import path of the overflowing functions based on the width. This way we can easily inspect cairo functions used in warp